www/src/components/docs-table-of-contents.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import { Link } from "gatsby"
+import { fontSizes, colors, space, mediaQueries } from "../utils/presets"
+
+function createItems(items, location) {
+  return (
+    items &&
+    items.map(item => (
+      <li>
+        <Link to={location.pathname + item.url}>{item.title}</Link>
+        {item.items && (
+          <ul css={{ marginLeft: space[6] }}>
+            {createItems(item.items, location)}
+          </ul>
+        )}
+      </li>
+    ))
+  )
+}
+
+function TableOfContents({ page, location }) {
+  return page.tableOfContents.items ? (
+    <div
+      css={{
+        background: colors.white,
+
+        [mediaQueries.xxl]: {
+          padding: space[6],
+          float: `right`,
+          marginLeft: space[10],
+          position: `sticky`,
+          top: 130,
+        },
+      }}
+    >
+      <h2
+        css={{
+          textTransform: `uppercase`,
+          fontSize: fontSizes[2],
```

Comment:
I'd go for `fontSizes[1]` here, and throw in a `letterSpacing: letterSpacings.tracked`—ref. the excellent https://practicaltypography.com/letterspacing.html for why we want extra letterspacing for all caps text.

Reply:
Done